pkg/lib/hooks.ts

```diff
@@ -262,10 +273,10 @@ export function useObject(create, destroy, deps, comps) {
 
     destroy_ref.current = destroy;
     useEffect(() => {
-        return () => destroy_ref.current?.(ref.current);
+        return () => { destroy_ref.current?.(ref.current!) };
```

Comment:
Why does it need to be wrapped into `{}` now? Does the current() call return something non-void?

Reply:
I think the `!` somehow caused syntactic problems if it wasn't in a block.  I honestly don't remember: it was a long time ago.